### ISSUE TITLE
geonames:install doesn't pass through database connection to all subsequent calls

### ISSUE THREAD

**Issue by karlshea** [michaeldrennen/Geonames#38]
Command: `php artisan geonames:install --connection=geonames --country=US --language=en`

Output:
```
The database connection name was set to: geonames
Testing database connection...
Confirmed database connection set up correctly.
GeoSetting::install() called on connection: geonames
...
alternate_names data was downloaded and inserted in 1393.3221490383 seconds.
Running the geonames:geoname artisan command in live mode.
Running geonames:download-geonames on database connection: mysql
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db.geonames_settings' doesn't exist (SQL: select * from `geonames_settings` limit 1)
```

However, running `php artisan geonames:install --connection=geonames --test` *does* work.

**Comment by CamaleonGamer** [comment]
I fixed this by adding the --conection flag to geonames: download-geonames in the Geoname.php file

$this->call('geonames:download-geonames', [**'--connection' => $this->connectionName**]);

I hope it helps you !!!

Camaleon

**Comment by karlshea** [comment]
Yes, that's exactly the fix I have in my PR (#39).
